receiver/riakreceiver/metadata.yaml

```
@@ -0,0 +1,70 @@
+name: riakreceiver
+
+attributes:
+  riak.node.name:
+    description: The name this node uses to identify itself.
+  request.operation:
```

```suggestion
  request:
```
Data point attribute names _tend_ to be shorter. In this case, `request` just seems like a more specific word for `operation`.